spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/TaskTemplate.java

```diff
@@ -165,12 +181,7 @@ public class TaskTemplate implements TaskOperations {
 			this.executionLaunchLink = null;
 		}
 
-		if (VersionUtils.isDataFlowServerVersionGreaterThanOrEqualToRequiredVersion(version, EXECUTIONS_CURRENT_RELATION_VERSION)) {
```

Comment:
Why did this move up? If its not required to be at the top, please keep it in place so as to avoid confusing during the review.

Reply:
I fixed the order to match the previous version.

Reply:
I would prefer change history over matching order in a different branch. Please undo the change.

Reply:
Will a squash into on commit not fix this so that the diff is smaller because the order is that same as before the PR branch?